src/settings.cpp

```diff
@@ -460,34 +460,28 @@ int32 IntSettingDesc::Read(const void *object) const
 }
 
 /**
- * Set the string value of a setting.
- * @param object The object the setting is to be saved in.
- * @param str The string to save.
+ * Validate the value of a setting.
```

Comment:
You cannot commit on commit messages in GitHub, so just a random place, sorry:

`Codechange: add helper functions to write an string setting value`

Is it me, or is this more like "Splitting Write_ValidateSetting into two functions" kinda thing?

btw, it is `a string`, not `an string` :)